packages/wpcom-checkout/src/get-introductory-offer-interval-display.ts

```diff
@@ -62,5 +64,45 @@ export function getIntroductoryOfferIntervalDisplay(
 			}
 		}
 	}
+	if ( remainingRenewalsUsingOffer > 0 ) {
+		text += ' - ';
+		if ( context === 'checkout' ) {
+			if ( remainingRenewalsUsingOffer === 1 ) {
+				text += String(
+					translate( 'The first renewal is also discounted.', {
```

Comment:
I might be wrong, but I think this logic is covered by the first translation string at L81. You can update that string to match this one (remove the placeholder) and then remove this block.

Reply:
The else block would definitely cover this case as well, however, that second block is actually intended to cover cases where the singular form is used for quantities larger than one (In some languages, the singular form is used for 21, 31, etc.) This is explained in the [i18n-calypso README](https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md#pluralization)

> It might seem redundant, but it is necessary for languages where a singular version may be used for counts other than 1.

I think that "The first 1 renewal is also discounted" wouldn't read as natural as just omitting the number one, so that's the reason I had to add this if clause to cover this particular case.